webpack.config.js

```diff
@@ -0,0 +1,47 @@
+
+const Encore = require('@symfony/webpack-encore');
+const Webpack = require('webpack');
+
+const path = require('path');
+Encore.reset();
+
+
+Encore
+    // the project directory where all compiled assets will be stored
+    .setOutputPath(`./bundle/Resources/public/build/`)
+
+    // the public path used by the web server to access the previous directory
+    .setPublicPath(`/bundles/netgenibexafieldtypehtmltext/build`)
+
+    .addEntry('app', `./bundle/Resources/js/app.js`)
+
+    .setManifestKeyPrefix('build/')
+
+    // empty the outputPath dir before each build
+    .cleanupOutputBeforeBuild()
+    .copyFiles([
+        { from: './node_modules/ckeditor4/', to: 'ckeditor/[path][name].[ext]', pattern: /\.(js|css)$/, includeSubdirectories: false },
```

Comment:
Why not use ckeditor 5?

Reply:
ckeditor5 now used, changed in https://github.com/netgen/ibexa-fieldtype-html-text/pull/3/commits/c93c56cfb6716a82db7e6f5b25d923edcc68027e